### DISCUSSION TITLE
Horizontal line between functions

### DISCUSSION THREAD

**Discussion by Shibohi** [godotengine/godot-proposals#8700] [Editor]
Hey,

if u have lage script files it gets messy for me to see where a function ends/begins.
It would be nice if we had a option like the "show_line_length_guideline" for lines between functions.

greetings

**Comment by aXu-AP** [comment]
I'm not sure how needed this feature is, but here's some tips that might help you to keep you oriented in big scripts.
* Keep eyes on the left side of the code when scrolling. When something is at the beginning of the line, it's certainly a new function.
* Style guide recommends to use 2 empty lines to separate functions.
* For big functions you can use folding.
* On the left side of the editor there's a panel with all functions in it, you can use it for navigation.

**Reply by Calinou** [reply]
You can also use code regions since Godot 4.2 to organize long functions or series of functions: https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_basics.html#code-regions
